### PR TITLE
Revamp the breadcrumb in the super organization and sub organizations

### PR DESCRIPTION
**Pull request by Shenali-SJ**
### Related Issues
- https://github.com/wso2/product-is/issues/17174

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx [hunk 1]
```diff
@@ -368,15 +370,24 @@ export const OrganizationSwitchBreadcrumb: FunctionComponent<OrganizationSwitchD
...
-            disable={ 
-                organizationConfigs.showSwitcherInTenants
-                    ? breadcrumbList?.length > 4
-                    : false
-            } 
-        />
+        <> 
+            { organizationConfigs.showSwitcherInTenants ? showTenantDropdown() :
+                (isSubOrg ? showTenantDropdown() : null) } 
+        </>
```
[the remark this answers is not in the document]
Comment (Shenali-SJ) [reply apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx]: ACK. This is fixed with the commit 7fd29cec34f0d44bff49d22879e6f41f539f5a8a

File: apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx [hunk 2]
```diff
@@ -75,6 +75,13 @@ export const OrganizationSwitchBreadcrumb: FunctionComponent<OrganizationSwitchD
         shouldSendRequest
     );
 
+    const isSubOrg: boolean = window[ "AppUtils" ].getConfig().organizationName;
+
+    const isShowSwitcher: boolean = (
+        (organizationConfigs?.showSwitcherInTenants && !isSubOrg) ||
```
Comment (dasuni-30) [inline apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx:81]: Shall we improve readability here?

 const isShowSwitcher: boolean = (
        (
            organizationConfigs?.showSwitcherInTenants
            && !isSubOrg
        ) || (
            !organizationConfigs?.showSwitcherInTenants
            && isSubOrg
        )

**Review by dasuni-30 — APPROVED** [review]

File: apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx (hunk 2 above)
Comment (dasuni-30) [inline apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx:81]: Shall we improve readability here?

 const isShowSwitcher: boolean = (
        (
            organizationConfigs?.showSwitcherInTenants
            && !isSubOrg
        ) || (
            !organizationConfigs?.showSwitcherInTenants
            && isSubOrg
        )

**Comment by dasuni-30** [comment]
Let's add the changeset and a screenshot.

**Comment by Shenali-SJ** [comment]
### Updated Breadcrumb
- In the super organization
Screenshot 2023-10-25 at 12 38 59

- In sub-organizations
Screenshot 2023-10-25 at 12 40 35

**Review by JayaShakthi97 — DISMISSED** [review]

File: apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx [hunk 3]
```diff
@@ -75,6 +75,11 @@ export const OrganizationSwitchBreadcrumb: FunctionComponent<OrganizationSwitchD
         shouldSendRequest
     );
 
+    const isSubOrg: boolean = window[ "AppUtils" ].getConfig().organizationName;
+
+    const isShowSwitcher: boolean = 
+    organizationConfigs?.showOrganizationDropdown || isSubOrg;
```
[the remark this answers is not in the document]
Comment (Shenali-SJ) [reply apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx]: This is resolved with 05279b7e82e0cc61e3f8390afc579c0d2d24e2a3

File: apps/console/src/features/tenants/components/dropdown/tenant-dropdown.tsx [hunk 4]
```diff
@@ -41,12 +37,9 @@ import {
     Dropdown,
     Grid,
     Icon,
-    Input,
     Item,
     Menu,
-    Placeholder,
-    SemanticICONS
-} from "semantic-ui-react";
+    Placeholder } from "semantic-ui-react";
```
[the remark this answers is not in the document]
Comment (Shenali-SJ) [reply apps/console/src/features/tenants/components/dropdown/tenant-dropdown.tsx]: This is resolved with 05279b7e82e0cc61e3f8390afc579c0d2d24e2a3

File: apps/console/src/features/tenants/components/dropdown/tenant-dropdown.tsx [hunk 5]
```diff
@@ -17,22 +17,18 @@
...
 } from "@wso2is/core/models";
 import { addAlert, setTenants } from "@wso2is/core/store";
 import {
     Button,
-    GenericIcon,
-    HeaderPropsInterface as ReusableHeaderPropsInterface
-} from "@wso2is/react-components";
-import { AxiosResponse } from "axios";
-import React, { FunctionComponent, ReactElement, ReactNode, SyntheticEvent, useEffect, useState } from "react";
+    GenericIcon } from "@wso2is/react-components";
```
[the remark this answers is not in the document]
Comment (Shenali-SJ) [reply apps/console/src/features/tenants/components/dropdown/tenant-dropdown.tsx]: This is resolved with 05279b7e82e0cc61e3f8390afc579c0d2d24e2a3

**Review by JayaShakthi97 — APPROVED** [review]

**Review by JayaShakthi97 — APPROVED** [review]

**Pull request merged**